.gitignore

```diff
@@ -1,9 +1,8 @@
 build
-
 cmake-build-release
```

Comment:
```suggestion
```
This can go away now, it is covered by the `*` version.

Reply:
Updated